src/utils/SystemDiagnostic.ts

```
@@ -0,0 +1,226 @@
+            return 25 // Fallback
+        }
+    }
+
+    private async measureSystemLoad(): Promise<number> {
+        try {
+            // Mesurer charge CPU simple
+            const { stdout } = await execAsync('top -bn1 | head -3')
+            
+            // Chercher "load average"
+            const loadMatch = stdout.match(/load average: ([\d.]+)/)
+            const load = loadMatch ? parseFloat(loadMatch[1]) : 0
```

Biome (1.9.4)

[error] Use Number.parseFloat instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseFloat instead.


(lint/style/useNumberNamespace)